HISTORY.md

```diff
@@ -14,6 +14,7 @@ _????-??-??_
  
  * Bump minimum Armadillo version to 10.8 (#3760).
 
+ * Adapt `NearestInterpolation` ANN layer to new Layer Inteface
```

Comment:
@andrewfurey21  could you add the number of the PR at the end of the line, like the above one ?